sdk/src/metrics/state/temporal_metric_storage.cc

```diff
@@ -69,7 +69,7 @@ bool TemporalMetricStorage::buildMetrics(CollectorHandle *collector,
                                                 instrument_descriptor_, aggregation_config_.get())
                                                 ->Merge(aggregation));
             merged_metrics->GetAllEnteries(
-                [](const MetricAttributes &attr, Aggregation &aggr) { return true; });
+                [](const MetricAttributes & /* attr */, Aggregation & /* aggr */) { return true; });
```

Comment:
nit - could you remove this statement (line 71,72) to iterate over the merged entries, as it is not doing anything. Seems to be part of debugging, and I forgot to remove. Thanks :)

Reply:
Done